recipes-nemomobile/mce/mce/20als-defaults.ini

```diff
@@ -0,0 +1,94 @@
+LevelsProfile8=4;8;12;16;20;24;28;32;36;40;44;50;56;61;67;72;78;84;89;95;100
+
+LimitsProfile9=1;2;4;6;11;19;34;61;109;195;350;404;466;538;621;716;826;954;1100;1269;1464
+LevelsProfile9=4;9;13;17;22;26;30;35;39;43;47;53;58;63;69;74;79;85;90;95;100
+
+LimitsProfile10=1;2;4;6;11;19;34;61;109;195;350;403;463;533;612;704;809;931;1070;1230;1414
+LevelsProfile10=5;10;14;19;23;28;32;37;41;46;50;55;60;65;70;75;80;85;90;95;100
+
+LimitsProfile11=1;2;4;6;11;19;34;61;109;195;350;402;460;527;604;692;793;908;1041;1193;1366
+LevelsProfile11=6;11;16;21;25;30;35;39;44;49;53;58;63;68;72;77;82;86;91;96;100
+
+LimitsProfile12=1;2;4;6;11;19;34;61;109;195;350;400;457;522;596;680;777;887;1013;1156;1320
+LevelsProfile12=7;12;17;22;27;32;37;42;47;52;56;61;65;70;74;78;83;87;92;96;100
+
+LimitsProfile13=1;2;4;6;11;19;34;61;109;195;350;399;454;516;587;669;761;866;985;1121;1275
+LevelsProfile13=8;14;19;24;29;34;39;44;49;54;59;64;68;72;76;80;84;88;92;96;100
+
+LimitsProfile14=1;2;4;6;11;19;34;61;109;195;350;397;451;511;579;657;745;845;958;1086;1231
+LevelsProfile14=10;16;21;26;31;36;42;47;52;57;62;66;70;74;78;81;85;89;93;97;100
+
+LimitsProfile15=1;2;4;6;11;19;34;61;109;195;350;396;447;506;571;646;730;824;932;1053;1189
+LevelsProfile15=11;17;22;28;33;38;44;49;55;60;65;69;72;76;79;83;86;90;93;97;100
+
+LimitsProfile16=1;2;4;6;11;19;34;61;109;195;350;395;444;500;564;635;715;805;906;1021;1149
+LevelsProfile16=13;19;24;30;35;41;46;52;57;63;68;72;75;78;81;84;88;91;94;97;100
+
+LimitsProfile17=1;2;4;6;11;19;34;61;109;195;350;393;441;495;556;624;700;786;882;989;1110
+LevelsProfile17=15;21;27;32;38;43;49;55;60;66;71;74;77;80;83;86;89;92;95;98;100
+
+LimitsProfile18=1;2;4;6;11;19;34;61;109;195;350;392;438;490;548;613;686;767;857;959;1072
+LevelsProfile18=18;24;30;35;41;46;52;58;63;69;74;77;80;82;85;87;90;93;95;98;100
+
+LimitsProfile19=1;2;4;6;11;19;34;61;109;195;350;391;435;485;541;602;671;748;834;929;1035
+LevelsProfile19=21;27;33;38;44;49;55;61;66;72;77;80;82;84;87;89;91;94;96;98;100
+
+LimitsProfile20=1;2;4;6;11;19;34;61;109;195;350;389;432;480;533;592;658;730;811;901;1000
+LevelsProfile20=25;31;36;42;47;53;58;64;69;75;80;82;84;86;88;90;92;94;96;98;100
+
+[BrightnessLed]
+
+LimitsProfile0=1;2;4;6;11;19;34;61;109;195;350;417;496;591;703;837;996;1186;1412;1681;2000
+LevelsProfile0=6;8;9;11;12;13;15;16;18;19;20;28;36;44;52;60;68;76;84;92;100
+
+[BrightnessKeypad]
+
+LimitsProfile0=1;2;3;6;11;20;36;66;121;220;489;599;732;896;1095;1340;1639;2005;2453;3000
+LevelsProfile0=30;33;36;39;42;45;48;51;54;57;64;68;72;76;80;84;88;92;96;100
+
+[BrightnessLPM]
+
+# Use Limits and Levels profiles to select the desired brightness for a given ALS and current brightness.
+# When the device doesn't have an ambient light sensor the default value for 400 will be used(i.e. 417(closest) => 28%)
+# Profiles can be added to have different LPM brightness levels at different input brightness levels.
+# This sets the maximum brightness to 28% in ambient mode.
```

Comment:
I cannot link the comment to the values below - copy paste error?

Reply:
This file is copied from the original: https://github.com/AsteroidOS/mce/pull/4/files#diff-705aa48c9445b9bdcc3d78edbff1b522R95

The values are just adjusted(improve battery life) to better suit smelt.

Reply:
Ah, so the comment does not add anything useful for smelt anymore?

Reply:
Correct

Reply:
Thanks for clarifying :)